FreeRTOS-Plus/Source/FreeRTOS-Plus-TCP/FreeRTOS_TCP_IP.c

```diff
@@ -1020,7 +1020,7 @@ uint32_t ulInitialSequenceNumber = 0;
 	case eARPCacheMiss:		/* An ARP table lookup did not find a valid entry. */
 	case eCantSendPacket:	/* There is no IP address, or an ARP is still in progress. */
 	default:
-		/* Count the number of times it couldn't find the ARP address. */
+		/* Count the number of times it could not find the ARP address. */
```

Comment:
This was fine before – the spell checker is not perfect and can't handle contractions.  You will see things like _wouldn_, _shouldn_, etc. in the lexicon.txt.